templates/details/_details-header.html

```diff
@@ -69,13 +69,13 @@ <h1 class="p-media-object__title">
     </div>
     <div class="col-4">
       <p style="margin-bottom: 0.5rem;"><small class="u-no-padding--top u-text--muted">Platform:</small></p>
-      {% if package.store_front.ubuntu_versions|length == 1 and package.store_front.ubuntu_versions[0] == "Kubernetes" %}
+      {% if package.store_front.platforms|length == 1 and package.store_front.platforms[0] == "Kubernetes" %}
```

Comment:
Here you should use: `channel_requested` instead of package

Reply:
It's the same Toto

Reply:
The store_front.platforms is generated with the channel_requested